### PR TITLE
Change send transaction

### PR DESCRIPTION
**Pull request by bordalix**
Related with https://github.com/vulpemventures/marina/pull/289

Previously `sendTransaction` returned the raw signed transaction.
Now blinds, sign and broadcast the transaction, and returns the transaction hash (id).

@tiero please review

### CODE REVIEW

File: src/pages/demo.tsx [hunk 1]
```diff
@@ -144,13 +143,13 @@ const MarinaExample: React.FC<Props> = () => {
               Send LBTC
             </button>
             {
-              tx.length > 0 &&
+              txHash.length > 0 &&
               <>
                 <p>
-                  {tx.substring(0, 20) + "..." + tx.substring((tx.length - 20), tx.length)}
+                  {txHash.substring(0, 20) + "..." + txHash.substring((txHash.length - 20), txHash.length)}
```
Comment (tiero) [inline src/pages/demo.tsx:149]: maybe now is fine to show it entirely since much more shorter

Comment (bordalix) [reply src/pages/demo.tsx:149]: Yes, and leave a safeguard in CSS in case it overflows on smaller displays.
On it.

Comment (bordalix) [reply src/pages/demo.tsx:149]: Forget it, is a demo 😄 
Doesn't need the CSS safeguard.

Comment (bordalix) [reply src/pages/demo.tsx]: Commit 1fd37ba fixes this.

**Pull request merged**